src/Conan.cmake

```diff
@@ -33,46 +33,52 @@ macro(run_conan)
     URL
     https://bincrafters.jfrog.io/artifactory/api/conan/public-conan)
 
-  # For multi configuration generators, like VS and XCode
-  if(NOT CMAKE_CONFIGURATION_TYPES)
-    message(STATUS "Single configuration build!")
-    set(LIST_OF_BUILD_TYPES ${CMAKE_BUILD_TYPE})
+  if(CONAN_EXPORTED)
+    # standard conan installation, in which deps will be defined in conanfile. It is not necessary to call conan again, as it is already running.
+    include(${CMAKE_BINARY_DIR}/../conanbuildinfo.cmake)
+    conan_basic_setup()
```

Comment:
Doesn't this need the arguments passed to conan_cmake_install?

Reply:
no, conan will handle this with the ``conan install`` part of the ``create`` command, it will use the __conan profile__ settings to do so.